pagopa/io-app
# fix: [PE-845] CGN Replace legacy error screen

## Short description
This pull request focuses on improving the error handling user interface in the `CgnDetailScreen` component. The main changes involve replacing the `GenericErrorComponent` with the `OperationResultScreenContent` component to provide a more detailed and visually appealing error screen.

## List of changes proposed in this pull request
- Updated the error handling logic to use `OperationResultScreenContent`, providing a pictogram, title, subtitle, and actions for retry and close operations

## How to test
- Tap on CGN card with no connection
- Check the outcome result
